pipeline/container/frontend/index.html

```diff
@@ -692,22 +919,10 @@
         const [dynamicFields, setDynamicFields] = React.useState([]);
         const [inputData, setInputData] = React.useState([]);
         const [pipeline, setPipeline] = React.useState(null);
+        const [outputData, setOutputData] = React.useState(null);
+        const [outputErrorData, setErrorOutputData] = React.useState(null);
 
-        React.useEffect(() => {
-          fetch(`${host}/v4/container/pipeline`)
-            .then((res) => res.json())
-            .then((data) => {
-              setPipeline(data);
-              setDynamicFields(
-                generateDynamicFieldsFromIOVariables(data.input_variables)
-              );
-            })
-            .catch((err) => console.log(err));
-        }, []);
-        console.log(pipeline);
-        React.useEffect(() => {
-          setInputData(generateInputDataArrayFromDynamicFields(dynamicFields));
-        }, [dynamicFields]);
```

Comment:
What handles the inputData updates now then? (When the form values change)

Reply:
you're right!

Reply:
re-added! good catch